src/modules/75-cf/strings/strings.en.yaml

```
@@ -530,14 +530,16 @@ onboarding:
   selectOrCreateFlag: Select or Create a Flag
   youCreated: You've created
   youreUsing: You're using
-  setupLabel: Set up your application and code to communicate with our platform.
+  selectEnvAndSdk: Select Environment and SDK
   selectLanguage: Select your language
-  selectYourEnvironment: Select your Environment
   createEnv: Create an environment
+  selectOrCreateEnvironment: Select or Create an Environment
+  environmentDescription: Environments are spaces within a project where you can independently configure Feature Flags. You can have multiple Environments and the same Flags can have a different state in each Environment.
   envCreated: Environment Created
-  sdkKeyLabel: Please create an SDK key for your environment
-  keyDescriptionServer: We generated a secret as the server-side SDK key of your new environment for you to use in your code below. This secret will be redacted once you leave the page, so make sure to copy and store your secret somewhere safe.
-  keyDescriptionClient: We generated a client-side SDK key of your new environment for you to use in your code below.
+  createSdkKey: Create an SDK Key
+  sdkKeyDescription: SDK keys are unique to each Environment and they authorize the connection between your application and the Feature Flag SDK Client.
+  keyDescriptionServer: We generated a secret as the server-side SDK key for <b>{{env}}</b> Environment for you to use in your code below. This secret will be redacted once you leave the page, so make sure to copy and store your secret somewhere safe.
+  keyDescriptionClient: We generated a client-side SDK key for <b>{{env}}</b> Environment. Use this in your code to authenticate the connection between your applicationand the Feature Flag SDK client.
```

```suggestion
  keyDescriptionClient: We generated a client-side SDK key for <b>{{env}}</b> Environment. Use this in your code to authenticate the connection between your application and the Feature Flag SDK client.
```
wee space missed here